src/mail-app/settings/MailSettingsViewer.ts

```diff
@@ -389,14 +389,16 @@ export class MailSettingsViewer implements UpdatableSettingsViewer {
 									}),
 								),
 						  ],
-					m(".h4.mt-l", lang.get("exportMailbox_label")),
-					this.mailExportController
-						? m(MailExportSettings, {
+					// export is only available on desktop
+					isDesktop() &&
```

Comment:
I would suggest that we shouldn't import `mailExportController` on non-desktop. If we move platform check where we ask for it from locator we might be able to achieve both things together.